service/src/test/resources/ehr/mapper/encountercomponents/expected-components-1.xml

```diff
@@ -455,6 +455,163 @@
                         </NarrativeStatement>
                     </component>
                     <component typeCode="COMP" contextConductionInd="true">
+                        <CompoundStatement classCode="CLUSTER" moodCode="EVN">
```

Comment:
The associated `DiagnosticReport` has 3 Specimens, 3 Test Results, and 3 filing comments. Each of the test results relates to one of the Specimens, and each Specimen has one test result. I can see no reason for a Dummy specimen to be produced.

Reply:
`expected-components-1.xml` has been reverted to main.